client/signup/steps/plans/tabbed-plans.js

```diff
@@ -0,0 +1,986 @@
+				),
+				planOne: {
+					included: false,
+					copy: translate( 'Not included' ),
+					mobileCopy: null,
+				},
+				planTwo: {
+					included: true,
+					copy: translate( '8,000 to choose from' ),
+					mobileCopy: translate( '8,000 themes to choose from' ),
+				},
+				planThree: {
+					included: true,
+					copy: translate( '8,000 to choose from' ),
+					mobileCopy: translate( '8,000 themes to choose from' ),
+				},
+			},
+			{
+				featureName: translate( 'eCommerce' ),
+				tooltip: translate(
+					'Build, manage, and scale your online store with unlimited products or services, multiple currencies, integrations with top shipping carriers, and eCommerce marketing tools.'
+				),
+				planOne: { included: false, copy: translate( 'Not included' ), mobileCopy: null },
+				planTwo: {
+					included: false,
+					copy: translate( 'Not included' ),
+					mobileCopy: null,
+				},
+				planThree: {
+					included: true,
+					copy: translate( 'Complete eCommerce platform' ),
+					mobileCopy: translate( 'Complete eCommerce platform' ),
+				},
+			},
+			{
+				featureName: translate( 'Custom domain name' ),
+				tooltip: translate(
+					'Remove “.wordpress.com” from your site address with a custom domain of your choosing. Free for one year (will renew at its regular price).'
+				),
+				planOne: {
+					included: true,
+					copy: translate( 'Free for the first year' ),
+					mobileCopy: translate( 'Free domain for the first year' ),
+				},
+				planTwo: {
+					included: true,
+					copy: translate( 'Free for the first year' ),
+					mobileCopy: translate( 'Free domain for the first year' ),
+				},
+				planThree: {
+					included: true,
+					copy: translate( 'Free for the first year' ),
+					mobileCopy: translate( 'Free domain for the first year' ),
+				},
+			},
+			{
+				featureName: translate( 'SEO' ),
+				tooltip: translate(
+					'SEO (Search Engine Optimization) helps your site rank higher, and more accurately in search engines, so more people can find your site.'
+				),
+				planOne: {
+					included: true,
+					copy: translate( 'Simple' ),
+					mobileCopy: translate( 'Simple SEO' ),
+				},
+				planTwo: {
+					included: true,
+					copy: translate( 'Professional for better ranking' ),
+					mobileCopy: translate( 'Professional SEO' ),
+				},
+				planThree: {
+					included: true,
+					copy: translate( 'Professional for better ranking' ),
+					mobileCopy: translate( 'Professional SEO' ),
+				},
+			},
+			{
+				featureName: translate( 'Customer support' ),
+				tooltip: translate(
+					'Keep moving forward with your site at any time. Our Happiness Engineers will help you via email, live chat, or 24/7 priority live chat, depending on the plan you choose.'
+				),
+				planOne: {
+					included: true,
+					copy: translate( 'Live chat and email' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **6** times:
`translate( 'Live chat and email support' )` *ES Score: 8*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/tabbed-plan-signup-test)